checkov/runner_filter.py

```diff
@@ -202,3 +202,66 @@ def is_policy_filtered(self, check_id: str) -> bool:
         if not self.filtered_policy_ids:
             return True
         return check_id in self.filtered_policy_ids
+    
+    def to_dict(self) -> Dict[str, Any]:
+        result: Dict[str, Any] = {}
```

Comment:
@achiar99 What happens if we add more attributes to the RunnerFilter? Who would enforce adding those here as well?
Is there a different way in python to grab all of the class attributes?

Reply:
i guess i can use `obj.__dict__` or `dir(obj)` instead